tox.ini

```diff
@@ -1,19 +0,0 @@
-[tox]
```

Comment:
@imran-ansari Can you check on this file, will this affect our unit test process, looks like some files are moved to different folder